mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/logging/impl/Log.java

```diff
@@ -739,4 +740,8 @@ SearchException infiniteRecursionForProjectionConstructor(
 					"Consider having just one setting enabled as after the index is recreated there is nothing to purge.")
 	void redundantPurgeAfterDrop();
 
+	@Message(id = ID_OFFSET + 121,
+			value = "Invalid ObjectPath encountered '%s'. See cause for details.")
+	SearchException invalidObjectPath(ObjectPath path, @Cause Exception cause);
```

Comment:
Please include the nested exception message like we do elsewhere:

```suggestion
			value = "Invalid ObjectPath encountered '%1$s': $2$s")
	SearchException invalidObjectPath(ObjectPath path, String causeMessage, @Cause Exception cause);
```

Two reasons for doing that:

1. Hibernate Search produces failure reports that aggregate all mapping failures in a tree. That failure report only includes the "outermost" exception, without stacktrace or the message of nested exceptions. So we need the "outermost" exception message to be self-sufficient.
2. A lot of Java users will not look at nested exceptions. Some novice users won't think of it, while some more experienced users just won't like to scroll down and get irritated.

Reply:
> won't like to scroll down and get irritated.

Hah, it's funny - I'm so used to scrolling and never even check the initial log entries as it's so rarely they have anything useful 😄